### ISSUE TITLE
Readme link to Python example is invalid

### ISSUE THREAD

**Issue by tzulberti** [sphinx-contrib/sqlalchemyviz#1]
The readme has the following links:

```
Python: | https://github.com/chintal/tendril/blob/master/tendril/utils/db.py#L201
```

But that file no longer exists so I can check what the variable was.

pd: This seems to be a great project.

**Comment by chintal** [comment]
Yes, this is true. I'll try to put up a self contained example here.

**Comment by chintal** [comment]
In the meanwhile, what was in the original link can be found here :

https://github.com/tendril-framework/tendril-utils-db/blob/41a229b4f9a57f37f4024a49f67d950e1cdbb03f/src/tendril/utils/db.py#L258

Unfortunately, this by itself does not actually generate output. You can take a look at the nearby get_metadata() as well for how it's supposed to work.
